README.md

```diff
@@ -141,17 +142,24 @@ $ cat my-policy.hcl | vault policy write my-policy -
 
 Vault tokens will be renewed by `secrets-manager` if the `ttl` is lower than `vault.max-token-ttl` and the token is renewable. But as per Vault's [documentation](https://www.vaultproject.io/docs/concepts/tokens.html#the-general-case), regular tokens will have their own max TTL that it's calculated on every renewal, so that a token will eventually expire. This can be ok for your use case, but for others a [periodic token](https://www.vaultproject.io/docs/concepts/tokens.html#periodic-tokens) could be much more convinient. In the case of a periodic token, the `period` will invalidate the `vault.renew-ttl-increment` option.
 
-To create a regular token attached to a policy:
 
-`$ vault token create -ttl=1h -policy=my-policy`
+### Vault AppRole
```

Comment:
I think it would be a good idea to add a comment at the top of the README (and probably in the release changelog) that this version introduces a breaking change with regard to the Vault auth method.

Reply:
Added to `CHANGELOG.md`